common/src/main/java/com/wynntils/features/user/QuestInfoOverlayFeature.java

```diff
@@ -96,5 +122,21 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
                             this.getRenderHorizontalAlignment(),
                             this.getRenderVerticalAlignment());
         }
+
+        @Override
+        public void renderPreview(PoseStack poseStack, float partialTicks, Window window) {
+            recalculatePreviewRenderTasks(); // we have to force update every time
+
+            FontRenderer.getInstance()
+                    .renderTextsWithAlignment(
+                            poseStack,
+                            this.getRenderX(),
+                            this.getRenderY(),
+                            Arrays.stream(toRenderPreview).toList(),
```

Comment:
Why is toRenderPreview not a List to begin with..?  (The same question goes for toRender, I assume)

Reply:
I guess I can change that up.